src/shrinkray/passes/clangdelta.py

```diff
@@ -126,9 +127,7 @@ async def query_instances(self, transformation: str, data: bytes) -> int:
                 ).stdout
             except subprocess.CalledProcessError as e:
                 msg = (e.stdout + e.stderr).strip()
-                if msg == b"Error: Unsupported file type!":
-                    raise ValueError("Not a C or C++ test case")
-                elif b"Assertion failed" in msg:
+                if b"Assertion failed" in msg:
```

Comment:
Might be worth adding a comment about why we ignore these (it's very common in clang_delta and considered harmless)